src/mam/mod_mam_muc_elasticsearch_arch.erl

```diff
@@ -79,7 +79,7 @@ archive_message(_Result, Host, #{message_id := MessageId,
                                  local_jid := RoomJid,
                                  remote_jid := FromJID,
                                  source_jid := SourceJid,
-                                 packet := Packet}) ->
+                                 packet := Packet} = Params) ->
```

Comment:
I think that we'd rarely want to log the packet to be honest, seems like bad for privacy 😕

Reply:
It's another topic :)
Elasticsearch for log messages usually has the same kind of protection as elasticsearch for MAM :D